packages/browser/src/index.ts

```diff
@@ -245,4 +281,22 @@ export default class LogtoClient {
       throw new LogtoClientError('invalid_id_token', error);
     }
   }
+
+  private saveCodeToken({
+    refreshToken,
+    idToken,
+    scope,
+    accessToken,
+    expiresIn,
+  }: CodeTokenResponse) {
+    localStorage.setItem(`${this.logtoStorageKey}:${refreshTokenKey}`, refreshToken);
+    localStorage.setItem(`${this.logtoStorageKey}:${idTokenKey}`, idToken);
+
+    this.refreshToken = refreshToken;
+    this.idToken = idToken;
```

Comment:
use property setter to write localStorage?

Reply:
I tried to implement using the property setter in different ways.

Plan A: keep the property also in the object
![image](https://user-images.githubusercontent.com/10594507/154524364-ed8c9e63-8d54-4b7d-ab15-faf6333198bc.png)

Plan B: get and set the property in the localStorage
![image](https://user-images.githubusercontent.com/10594507/154524272-77ea6a75-52a0-4787-83db-53fdcb1ab635.png)

Plan C: Stay the same?

Plan D: TODO

@logto-io/eng any idea?

Reply:
Use Plan A. Updated. PTAL @gao-sun .

See [discussion](https://silverhand-io.slack.com/archives/C02A8G4HVAM/p1645118392840729) on Slack.